src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/TestFactory.cs

```diff
@@ -349,7 +367,37 @@ private void AddHackerUser(List<ProCoSysProject> commonProCoSysProjects)
                         new TestProfile
                         {
                             FullName = "Sigurd Signer",
-                            Oid = _signerOid
+                            Oid = SignerOid
+                        },
+                    ProCoSysPlants = commonProCoSysPlants,
+                    ProCoSysPermissions = new List<string>
+                    {
+                        Permissions.COMMPKG_READ,
+                        Permissions.MCPKG_READ,
+                        Permissions.PROJECT_READ,
+                        Permissions.LIBRARY_FUNCTIONAL_ROLE_READ,
+                        Permissions.USER_READ,
+                        Permissions.IPO_READ,
+                        Permissions.IPO_SIGN,
+                        Permissions.IPO_WRITE,
+                        Permissions.IPO_ATTACHFILE,
+                        Permissions.IPO_DETACHFILE
+                    },
+                    ProCoSysProjects = commonProCoSysProjects
+                });
+
+        // Authenticated user with necessary permissions to COMPLETE invitations
+        private void AddCompleterUser(
```

Comment:
No need for both Completer and Accepter user. They have same permissions. Endpoints for Accept and Complete both need 2 permissions: SIGN + WRITE, and can therefore be tested with same user-object

Reply:
Each invitation is required to have a contractor and a construction company and these will never be the same person. Since we have a validation rule checking if current user oid=the person completing/accepting oid, a test user needs to = a participant. For the sake of keeping with the accurate structure and business rules, I'd like to keep as is.

Reply:
Nah ... unit tests should cover that scenario. We can have a chat